packages/console/src/App.module.scss

```diff
@@ -1,4 +1,9 @@
 .skeleton {
+
+  // TO-DO: move to theme component
+  --color-on-surface-variant: #47464e;
+  --color-primary: #4f37f9;
+  --color-primary-90: #e5dfff;
```

Comment:
What does `90` stand for?

Reply:
grayscale/alpha

Reply:
name from figma, will update if needed

Reply:
ok, then I'm good with it, as long as it's consistent with figma 👌